simcal/jwst_tests/imaging/test_nircam.py

```diff
@@ -6,6 +6,15 @@
 from jwst.pipeline import Detector1Pipeline
 from jwst.pipeline import Image2Pipeline
 
+from ci_watson.artifactory_helpers  import (check_url, get_bigdata_root,
+    get_bigdata,
+    BigdataError,
+)
+
+
+data_root = get_bigdata_root()
```

Comment:
Since we only need `get_bigdata`, I would remove the other imports.  Also, we don't need `data_root`, so that line can be removed too.